content/types/_http_paginated_response.textile

```diff
@@ -43,29 +44,31 @@ bq(definition).
   csharp:   Boolean HasNext()
   java:     Boolean hasNext()
   swift,objc: Boolean hasNext()
+  go:       HasNext() (bool)
```

Comment:
I don't see this method in Go, https://github.com/ably/ably-go/search?q=hasnext&unscoped_q=hasnext